### ISSUE TITLE
InputfieldRadios forces entity encoded labels regardless of entityEncodeText setting

### ISSUE THREAD

**Issue by Toutouwai** [processwire/processwire-issues#1582]
### Short description of the issue

Sometimes it's useful to be able to add extra markup to the labels in an inputfield. This is particularly the case when using the Inputfields API for front-end forms, and this also applies to the FormBuilder module.

In this example I have a Page Reference field that's using InputfieldCheckboxes, and I want to add some extra markup including a note from a field on the selectable pages.

```php
$wire->addHookAfter('InputfieldCheckboxes::renderReadyHook', function (HookEvent $event) {
    $inputfield = $event->object;
    if($inputfield->name !== 'animal') return;
    foreach($inputfield->getOptions() as $id => $label) {
        $p = $event->wire()->pages->get($id);
        $label = "<b>$p->title</b>";
        $label .= "<div class='summary'>$p->summary</div>";
        $inputfield->optionLabel($id, $label);
    }
});
```

This first attempt doesn't work as hoped because the HTML tags get entity encoded.

![2022-06-08_195517](https://user-images.githubusercontent.com/1538852/172563059-975edcb6-7803-440b-8b4f-e7f16a869cfd.png)

But that's fixable by setting the inputfield `entityEncodeText` option to false.

```php
$wire->addHookAfter('InputfieldCheckboxes::renderReadyHook', function (HookEvent $event) {
    $inputfield = $event->object;
    if($inputfield->name !== 'animal') return;
    $inputfield->entityEncodeText = false;
    foreach($inputfield->getOptions() as $id => $label) {
        $p = $event->wire()->pages->get($id);
        $label = "<b>$p->title</b>";
        $label .= "<div class='summary'>$p->summary</div>";
        $inputfield->optionLabel($id, $label);
    }
});
```

![2022-06-08_195501](https://user-images.githubusercontent.com/1538852/172563100-2483d57c-b5b4-4d71-8566-e4e014bf5290.png)

But if I switch to InputfieldRadios then the labels are still entity encoded and there seems to be no way to stop this happening without modifying the core module. 

```php
$wire->addHookAfter('InputfieldRadios::renderReadyHook', function (HookEvent $event) {
    $inputfield = $event->object;
    if($inputfield->name !== 'animal') return;
    $inputfield->entityEncodeText = false;
    foreach($inputfield->getOptions() as $id => $label) {
        $p = $event->wire()->pages->get($id);
        $label = "<b>$p->title</b>";
        $label .= "<div class='summary'>$p->summary</div>";
        $inputfield->optionLabel($id, $label);
    }
});
```

![2022-06-08_200550](https://user-images.githubusercontent.com/1538852/172565301-298d40d0-c97b-4691-ae4e-3057b5947be6.png)

Sidenote: the way that InputfieldRadios replaces spaces with `<wbr>` tags [by default](https://github.com/processwire/processwire/blob/3acd7709c1cfc1817579db00c2f608235bdfb1e7/wire/modules/Inputfield/InputfieldRadios/InputfieldRadios.module#L36) seems strange to me. Not sure what the reason is but if it is for presentation purposes then CSS would be better. This setting [is turned off by AdminThemeUikit](https://github.com/processwire/processwire/blob/3acd7709c1cfc1817579db00c2f608235bdfb1e7/wire/modules/AdminTheme/AdminThemeUikit/init.php#L47-L49) but if InputfieldRadios is used in the front-end (including by FormBuilder) and you don't want these unexpected tags then you have to override the default and it's not obvious that you would need to look to `$config` settings to do this. Could this setting be turned off by default and solved by CSS, or turned on by legacy themes only if needed?

### Expected behavior

InputfieldCheckboxes and InputfieldRadios seem similar in a lot of ways so I expected them both to support the same options for control of entity encoding in labels.

### Suggestion for a possible fix

Only entity encode labels if entityEncodeText is true, similar to how it is handled in InputfieldCheckboxes.

```php
if($this->entityEncodeText) $label = $this->entityEncode($label, Inputfield::textFormatBasic);
```

### Setup/Environment

* ProcessWire version: 3.0.200

**Comment by ryancramerdesign** [comment]
Thanks @Toutouwai I've pushed an update for InputfieldRadios so that it is handled the same as InputfieldCheckboxes. That was the original intention but I think it had just never come up. As for the `<wbr>` tags, those were added to solve some issue someone was having at one point, but I don't think it's something that came up for me and I don't really remember what the point was. Maybe it was to resolve some odd browser behavior at the time, which might not even be applicable anymore. I have set the default for those wbr tags to be disabled now.
